digdag-docs/src/operators.rst

```diff
@@ -870,3 +870,81 @@ embulk>: Embulk data transfer
 
   * :command:`embulk>: embulk/mysql_to_csv.yml`
 
+
+s3_wait>: Wait for a file in Amazon S3
+--------------------------------------
+
+The **s3_wait>:** operator waits for file to appear in Amazon S3.
+
+
+.. code-block:: yaml
+
+    +wait:
+      s3_wait>: my-bucket/my-key
+
+Secrets
+~~~~~~~
+
+:command:`aws.access-key`
+  The AWS Access Key ID to use when accessing S3.
+
+:command:`aws.secret-key`
+  The AWS Secret Access Key when accessing S3.
+
+:command:`aws.sse-key`
+  An optional Server-Side Encryption key to use when accessing S3. Must be Base64 encoded.
```

Comment:
Why don't you use `customer provided Server-Side Encryption key` here so that users know it's for SSE-C not SSE-S3?